eslint.config.js

```diff
@@ -1,12 +1,26 @@
 import jsdoc from "eslint-plugin-jsdoc";
+import eslintjs from "@eslint/js";
+
+const {configs: eslintConfigs} = eslintjs;
```

Comment:
```suggestion
import {configs: eslintConfigs} from "@eslint/js";
```

Not possible?

Reply:
Not possible, it ships as a commonjs module and node tells me to do it this way if I try use a named import 

I could try again but doubt it'll have changed. Node would throw and told me it's because it uses a default cjs export

Reply:
Makes sense!